### PR TITLE
CORTX-29720 Remove deprecated keys from config files - Service framework

### PR DESCRIPTION
**Pull request by zahidsk**
Signed-off-by: Zahid Shaikh <zahid.shaikh@seagate.com>

### CODE REVIEW

File: k8_cortx_cloud/solution.yaml [hunk 1]
```diff
@@ -24,11 +24,9 @@ solution:
     rancher: ghcr.io/seagate/local-path-provisioner:v0.0.20
     busybox: ghcr.io/seagate/busybox:latest
   common:
-    setup_size: large
```
Comment (tanujashinde0405) [inline k8_cortx_cloud/solution.yaml:27]: why we are removing setup_size from solution.yaml and not from _config.tpl

**Review by tanujashinde0405 — CHANGES_REQUESTED** [review]

**Review by kailashyadav11 — APPROVED** [review]

File: k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/_config.tpl [hunk 2]
```diff
@@ -17,7 +17,6 @@ cortx:
     release:
       name: CORTX
       version: {{ .Values.cortxVersion }}
-    environment_type: K8                                            # DEPRECATED
     setup_size: {{ .Values.cortxSetupSize }}
```
Comment (tanujashinde0405) [inline k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/_config.tpl:20]: this also needs to be removed right?

**Review by tanujashinde0405 — CHANGES_REQUESTED** [review]

File: k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/_config.tpl (hunk 2 above)
Comment (zahidsk) [reply k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/_config.tpl]: removed

**Review by tanujashinde0405 — CHANGES_REQUESTED** [review]
also remove these keys from values.yaml and deploy script also.

File: k8_cortx_cloud/solution.yaml (hunk 1 above)
Comment (zahidsk) [reply k8_cortx_cloud/solution.yaml:27]: Removed from both the places

**Review by keithpine — CHANGES_REQUESTED** [review]
solution.example.yaml file also needs to be updated.

**Comment by zahidsk** [comment]
> solution.example.yaml file also needs to be updated.

updated in solution.example.yaml file

**Comment by zahidsk** [comment]
Updated workign ghcr build 2.0.0-707 in solution.yaml and solution.example.yaml files

**Review by tanujashinde0405 — APPROVED** [review]

File: k8_cortx_cloud/solution.example.yaml [hunk 3]
```diff
@@ -11,22 +11,20 @@ solution:
...
     kafka: ghcr.io/seagate/kafka:3.0.0-debian-10-r7
     zookeeper: ghcr.io/seagate/zookeeper:3.8.0-debian-10-r9
     rancher: ghcr.io/seagate/local-path-provisioner:v0.0.20
     busybox: ghcr.io/seagate/busybox:latest
   common:
-    setup_size: large
     storage_provisioner_path: /mnt/fs-local-volume
     container_path:
       local: /etc/cortx
-      shared: /share
```
[the remark this answers is not in the document]
Comment (zahidsk) [reply k8_cortx_cloud/solution.example.yaml:29]: Updated values.yaml and solution_stub.yaml files 
Removed and tested

File: k8_cortx_cloud/solution.yaml [hunk 4]
```diff
@@ -13,22 +13,20 @@ solution:
...
-    cortxserver: ghcr.io/seagate/cortx-rgw:2.0.0-689
-    cortxha: ghcr.io/seagate/cortx-all:2.0.0-689
-    cortxclient: ghcr.io/seagate/cortx-all:2.0.0-689
-    consul: ghcr.io/seagate/consul:1.11.4
+    cortxcontrol: ghcr.io/seagate/cortx-all:2.0.0-707
+    cortxdata: ghcr.io/seagate/cortx-all:2.0.0-707
+    cortxserver: ghcr.io/seagate/cortx-rgw:2.0.0-707
+    cortxha: ghcr.io/seagate/cortx-all:2.0.0-707
+    cortxclient: ghcr.io/seagate/cortx-all:2.0.0-707
+    consul: ghcr.io/seagate/consul:1.10.0
```
[the remark this answers is not in the document]
Comment (zahidsk) [reply k8_cortx_cloud/solution.yaml]: Updated

**Review by keithpine — APPROVED** [review]
Looks good, thanks!

**Pull request merged**